src/main/webapp/script.js

```diff
@@ -36,46 +36,69 @@ function initCommentSlider() {
 }
 
 /**
- * Retreive the youtube comments from the url.
+ * Retrieve the youtube comments given the URL
+ * @param {String} url to pass into backend
+ * @return {JSON} comments json object of comment objects
  */
-async function getYouTubeComments() {
-  const urlInput = document.getElementById('link-input');
-  const url = extractYouTubeUrl(urlInput.value);
+async function getYouTubeComments(url) {
+  url = cleanseUrl(url);
   const numComments = document.getElementById(SLIDER_NAME).value;
-  const response =
-      await fetch('/YouTubeComments?url='+url+'&numComments='+numComments);
+  const response = await fetch('/YouTubeComments?url=' + url +
+    '&numComments=' + numComments);
   const comments = await response.json();
   return comments;
 }
 
 /**
- * Fetches data and adds to html
+ * Wrapper function for preparing onClick function
  */
-async function getChart() {
-  $('form').submit(async function() {
-    document.getElementById('loading-img').style.display = 'block';
-    commentStats = await getYouTubeComments();
-    sentimentBucketList = commentStats.sentimentBucketList;
-    wordFrequencyMap = commentStats.wordFrequencyMap;
-
-    displaySentimentBucketChart(sentimentBucketList);
-    displayWordCloudChart(wordFrequencyMap);
-    averageScore = commentStats.averageScore;
-    const averageContainer = document.getElementById('average-score-container');
-    averageContainer.innerHTML = 'Average Sentiment Score: ' + averageScore;
+function onButtonPress() {
+  $('.submitBtn').click(function() {
+    showLoadingGif();
+    getChart();
   });
 }
 
+/**
+ * Fetches data and adds to html
+ * @param {String} url youtube url to retrieve comments
+ */
+async function getChart(url) {
```

Comment:
This method doesn't just show chart information, it shows a lot other information as well now. Update the name?